### PR TITLE
Revert change for windows cert pool access in 1.8

### PR DESCRIPTION
**Pull request by dmcgowan**
The upstream change to allow access to the windows system cert pool was reverted, reverting and updating messaging. Maybe 1.9....https://github.com/golang/go/issues/18609

Thanks @cyli for catching this!

ping @aaronlehmann

### CODE REVIEW

**Comment by aaronlehmann** [comment]
LGTM

File: tlsconfig/certpool_go17.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 func SystemCertPool() (*x509.CertPool, error) {
 	certpool, err := x509.SystemCertPool()
 	if err != nil && runtime.GOOS == "windows" {
-		logrus.Infof("Unable to use system certificate pool: %v", err)
+		logrus.Infof("Unable to use system certificate pool with custom certificate pool: %v", err)
```
Comment (thaJeztah) [inline tlsconfig/certpool_go17.go]: Bit unsure about the error

Comment (dmcgowan) [inline tlsconfig/certpool_go17.go]: The wording or the assumption that is used for a custom certificate pool?

Comment (thaJeztah) [inline tlsconfig/certpool_go17.go]: The wording is a bit confusing "use system pool with custom pool". Is the warning that they cannot be combined? (sorry can't fully grasp what it's saying :smile:)

**Comment by dmcgowan** [comment]
Removed updated info message, whatever messaging is needed can be done by Docker. Added an empty check on the server side before calling that function to prevent setting an empty pool instead of nil.

**Pull request merged**